arch/arm/soc/nordic_nrf5/nrf52/power.c

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <soc_power.h>
+#include <nrf_power.h>
+
+/* System_OFF is deepest Power state available, On exiting from this
+ * state CPU including all peripherals reset
+ */
+static void _system_off(void)
```

Comment:
This function becomes unused when `CONFIG_SYS_POWER_DEEP_SLEEP` is not defined.
Thus, it should be surrounded with `#if (defined(CONFIG_SYS_POWER_DEEP_SLEEP))`.